client/src/pages/WhatIsAutism/WhatIsAutism.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import ReactPlayer from "react-player/youtube";
+import { BsPlayCircleFill } from "react-icons/bs";
+import whatIsAutismImg from "../../assets/images/what-is-autism.jpeg";
+import whatIsAutismVideoImg from "../../assets/images/what-is-autism-video.jpg";
+import FooterNav from "../Home/FooterNav/FooterNav";
+import FooterIcon from "../Home/FooterIcon/FooterIcon";
+import FooterLogo from "../Home/FooterLogo/FooterLogo";
+import "./WhatIsAutism.css";
+
+function WhatIsAutism() {
+	return (
+		<div className="what-is-autism-container">
```

Comment:
you need change the way you split the whole layout into divs, i'll share image with you.